test/basic/RequestLinkXChain.test.ts

```diff
@@ -0,0 +1,78 @@
+import { ethers } from 'ethersv5'
+import peanut, { getDefaultProvider, getSquidRouterUrl, signAndSubmitTx } from '../../src/index' // import directly from source code
+import dotenv from 'dotenv'
+import { describe, it } from '@jest/globals'
+import { BigNumber } from 'ethersv5'
+import { EPeanutLinkType } from '../../src/consts/interfaces.consts'
+dotenv.config()
+
+describe('Peanut XChain request links fulfillment tests', function () {
+	describe('Peanut XChain request link fulfillment tests', function () {
+		it('Create a request link and fulfill it cross-chain', async function () {
+			peanut.toggleVerbose(true)
+			const userPrivateKey = process.env.TEST_WALLET_X_CHAIN_USER!
+			// const relayerPrivateKey = process.env.TEST_WALLET_X_CHAIN_RELAYER!
+
+			// Parameters that affect the test behaviour
+			const sourceChainId = '10'
+			const destinationChainId = '137'
+			const amountToTestWith = 0.1
+			const tokenAddress = '0x0b2C639c533813f4Aa9D7837CAf62653d097Ff85'
+			const tokenDecimals = 6
+			const APIKey = process.env.PEANUT_DEV_API_KEY!
+			const sourceChainProvider = await getDefaultProvider(sourceChainId)
+			console.log('Source chain provider', sourceChainProvider)
```

Comment:
Quesstion: what token/chain is this? what if the person running this test doesnt have that balance, how does he get it?